packages/expo-image-picker/android/src/main/java/expo/modules/imagepicker/tasks/ImageResultTask.kt

```diff
@@ -108,16 +98,17 @@ open class ImageResultTask(
   private fun readExif() = Bundle().apply {
     contentResolver.openInputStream(uri)?.use { input ->
       val exifInterface = ExifInterface(input)
-      exifTags.forEach { (type, name) ->
-        if (exifInterface.getAttribute(name) != null) {
+      EXIF_TAGS.flatMap { (type, tags) -> tags.map { tag -> type to tag } }
```

Comment:
Hmm, this line is very magical 🧙‍♀️

From the context, it is very easy to know what it's doing, but I would put it inside the helper function to make it more readable.